tests/ballerina-unit-test/src/test/resources/test-src/streaming/streamingv2-unique-length-window-test2.bal

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/runtime;
+import ballerina/streams;
+
+type Teacher record {
+    string name;
+    int age;
+    string status;
+    string school;
+};
```

Comment:
Do these records need to be open ones?

Reply:
@pubudu91 I did not get that. Can you please explain

Reply:
@LahiruJayasekara I had a chat with Pubudu on this. It seems we haven't considered the open/closed record things for streaming impl. Let's consider that separately.

Reply:
all right